docs/mast/mast.rst

```diff
@@ -1,3 +1,5 @@
+.. doctest-skip-all
```

Comment:
I'm afraid you need to revert most of these docs changes, they were mostly came from https://github.com/astropy/astroquery/pull/1975 that enabled the docs being tested.